script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
Consider using a context manager (`with`) with `json.load()` to handle file reading and JSON parsing in a cleaner manner. It simplifies the code and is less error-prone compared to manually reading files and parsing.
```suggestion
with open('scripts/example_data.json', 'r') as json_file:
    print(json.dumps(json.load(json_file), indent=2))
```